extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1141,10 +1141,11 @@ export class AmpStoryPage extends AMP.BaseElement {
           .addEventListener('click', () => this.openAttachment());
 
       const textEl = this.openAttachmentEl_
-          .querySelector('.i-amphtml-story-page-open-attachment-text');
+          .querySelector('.i-amphtml-story-page-open-attachment-label');
 
-      const openAttachmentLabel = Services.localizationService(this.win)
-          .getLocalizedString(
+
+      const openAttachmentLabel = attachmentEl.getAttribute('data-cta-label') ||
```

Comment:
Optional: Maybe we should keep the `-text` suffix instead of `-label`, to align with `data-tooltip-text`?

Reply:
Done